### PR TITLE
- Add testing method for packages

### PR DESCRIPTION
**Pull request by guillep**
Some enhancements

- Add testing method for packages
- Add organization in snapshot

### CODE REVIEW

File: src/Ring2-Compatibility.package/RG2MethodDefinition.extension/class/className.selector.isMetaSide..st [hunk 1]
```diff
@@ -0,0 +1,12 @@
+*Ring2-Compatibility
+className: aString selector: aString2 isMetaSide: isMeta
```
Comment (guillep) [inline src/Ring2-Compatibility.package/RG2MethodDefinition.extension/class/className.selector.isMetaSide..st:2]: This change is strange... I don't understand why it is here, I did not change this method

File: src/Ring2-Compatibility.package/Metaclass.extension/instance/asRing2Definition.st [hunk 2]
```diff
@@ -1,4 +1,6 @@
 *Ring2-Compatibility
 asRing2Definition
 
-	^ self theNonMetaClass asRing2Definition theMetaClass
\ No newline at end of file
+	^ (self theNonMetaClass asRing2Definition theMetaClass)
+		setNameFromSiblingIfNeeded;
```
Comment (guillep) [inline src/Ring2-Compatibility.package/Metaclass.extension/instance/asRing2Definition.st:5]: Same here, I did not change this method

**Review by guillep — COMMENTED** [review]
Pavel, there are some strange things, can you check if the PR is ok?

File: src/Ring2-Compatibility.package/RG2MethodDefinition.extension/class/className.selector.isMetaSide..st (hunk 1 above)
Comment (pavel-krivanek) [reply src/Ring2-Compatibility.package/RG2MethodDefinition.extension/class/className.selector.isMetaSide..st:2]: I was modifying the branch with with code for old Ring compatibility

**Pull request merged**